com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -1135,29 +1138,42 @@ RegId AddBricks(List<Brick> bricks, ProbeBrickPool.DataLocation dataloc, ProbeBr
             m_Registry.Add(id, ch_list);
 
             // Build index
-            m_Index.AddBricks(id, bricks, ch_list, m_Pool.GetChunkSize(), m_Pool.GetPoolWidth(), m_Pool.GetPoolHeight(), cellUpdateInfo);
+            m_Index.AddBricks(id, bricks, ch_list, ProbeBrickPool.GetChunkSize(), m_Pool.GetPoolWidth(), m_Pool.GetPoolHeight(), cellUpdateInfo);
+
+            // Update CellInfo
+            cellInfo.regId = id;
+            cellInfo.updateInfo = cellUpdateInfo;
+            cellInfo.chunkList = ch_list;
+            cellInfo.loaded = true;
+
+            // Update indirection buffer
+            m_CellIndices.UpdateCell(cellInfo.flatIdxInCellIndices, cellUpdateInfo);
```

Comment:
It is a bit weird here that we are updating cell info and the cell only indirection buffer in something called AddBricks, I'd prefer if we have this Cell level changes outside.

Reply:
It's mostly because iirc it was already kinda split like that. What I did was move back pool update in the same function AddBrick. I still intend to clean this further at some point as it's a bit messy right now. Also I'd like to find a better name than "AddBricks" because IMO it does not really convey what this does (aka, allocate and update indices and probe sh data).

Reply:
Before the cell level stuff was done in LoadingPendingCell _after_ the add bricks. 

And yes the name is not great indeed, but let's keep it for this PR and change it after.